<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>733</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,10,0,0,0,0,5" columnstretch="10,1,10">
      <item row="1" column="0">
       <widget class="QTextEdit" name="chatText">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QCheckBox" name="analyzeSentimentCheck">
          <property name="text">
           <string>Sentiment</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="classifyTopicCheck">
          <property name="text">
           <string>Classify topic</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QComboBox" name="topicPlotCombo">
          <item>
           <property name="text">
            <string>M&amp;S topics</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>LDA topics</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="resetButton">
          <property name="text">
           <string>Reset</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="enterButton">
          <property name="text">
           <string>Enter</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <widget class="QLineEdit" name="inputQueryText"/>
      </item>
      <item row="6" column="0" colspan="3">
       <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="5,1,1">
        <item>
         <widget class="QTableWidget" name="topicTable">
          <column>
           <property name="text">
            <string>Num.</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Date</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Agent</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>M&amp;S topic</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>LDA topic</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Level factor</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="levelFactorText">
              <property name="text">
               <string>1.0</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Chatbot version</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="versionCombo">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <item>
               <property name="text">
                <string>Select</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>V0</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>V1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>V2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>V3</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="saveModelButton">
            <property name="text">
             <string>Save model</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Nearest user input</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nearestInputText"/>
        </item>
       </layout>
      </item>
      <item row="1" column="1" rowspan="4" colspan="2">
       <widget class="QWidget" name="PlotWidget" native="true">
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <layout class="QVBoxLayout" name="PlotLayout"/>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>871</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
